animation.js

```diff
@@ -15,6 +17,8 @@ function Animation (opts) {
   this.prevTime = Date.now() // ms
   this.channels = opts.channels || []
   this.changed = new Signal()
+  this.currentOutputVec3 = vec3.create()
```

Comment:
Instead of attaching it via this, I would extract it at the top of the file as a let because it is only used for the updates..